bin/kubernetes/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: metabase
+  namespace: metabase
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "gce"
+spec:
+  tls:
+  - hosts:
+    - metabase.vikramtiwari.com
```

Comment:
I should have put in proper templating. I was testing this for my website. For your question, yes, this will be used as a template.